### DISCUSSION TITLE
ModuleNotFoundError while importing gseapy

### DISCUSSION THREAD

**Discussion by RainHan2001** [zqfang/GSEApy#164] [Q&A] [answered]
Hi,

I would like to seek advice for the error encountered when importing gseapy, it was installed these days and is the latest version. Yet I met the error message below:

***
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-7-e90d64dc96b1> in <module>
----> 1 import gseapy as gp

c:\Users\RainHan\anaconda3\lib\site-packages\gseapy\__init__.py in <module>
      7 from .biomart import Biomart
      8 from .enrichr import Enrichr
----> 9 from .gsea import GSEA, Prerank, Replot, SingleSampleGSEA
     10 from .parser import get_library_name
     11 from .plot import barplot, dotplot, gseaplot, heatmap

c:\Users\RainHan\anaconda3\lib\site-packages\gseapy\gsea.py in <module>
     13 
     14 from gseapy.base import GSEAbase
---> 15 from gseapy.gse import Metric, gsea_rs, prerank_rs, ssgsea_rs  # import gseapy rust lib
     16 from gseapy.parser import gsea_cls_parser
     17 from gseapy.plot import gseaplot

ModuleNotFoundError: No module named 'gseapy.gse'

***
Is there a way to solve this?

Thank you for advice!

**Comment by zqfang** [comment]
Yeah, please refer to #161. gseapy nowadays needs rust compiler.

If you use windows, you have to  `pip install  gseapy`.

**Comment by zqfang** [comment, marked as answer]
The newest release shipped with pre-compiled binaries for the most common platforms.

just use `pip install gseapy`. No more installation of rust compiler
